.github/workflows/firefox.yaml

```diff
@@ -0,0 +1,42 @@
+name: Firefox
```

Comment:
Why is this a separate workflow? Can't it be run with the other local tests?

Reply:
done!